src/engine/CursorLoader.cs

```diff
@@ -0,0 +1,13 @@
+using Godot;
+
+/// <summary>
+///   Replaces default cursor shapes with custom made
+/// </summary>
```

Comment:
Remarks should be added here that the default custom cursor is set in project settings.